python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,127 @@
+* [IntelliJ](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [VS Code](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/welcome.html)
+* [Visual Studio](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/welcome.html)
+
+## Deploy the sample application
+
+The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. It uses Docker to run your functions in an Amazon Linux environment that matches Lambda.
+
+To use the SAM CLI, you need the following tools.
+
+* AWS CLI - [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and [configure it with your AWS credentials].
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+{%- if cookiecutter.runtime == 'python2.7' %}
+* [Python 2.7 installed](https://www.python.org/downloads/)
+{%- else %}
+* [Python 3 installed](https://www.python.org/downloads/)
+{%- endif %}
+* Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
+
+The SAM CLI uses an Amazon S3 bucket to store your application's deployment artifacts. If you don't have a bucket suitable for this purpose, create one. Replace `BUCKET_NAME` in the commands in this section with a unique bucket name.
+
+```bash
+{{ cookiecutter.project_name }}$ aws s3 mb s3://BUCKET_NAME
+```
+
+To prepare the application for deployment, use the `sam package` command.
+
+```bash
+{{ cookiecutter.project_name }}$ sam package \
+    --output-template-file packaged.yaml \
+    --s3-bucket BUCKET_NAME
+```
+
+The SAM CLI creates deployment packages, uploads them to the S3 bucket, and creates a new version of the template that refers to the artifacts in the bucket. 
+
+To deploy the application, use the `sam deploy` command.
+
+```bash
+{{ cookiecutter.project_name }}$ sam deploy \
+    --template-file packaged.yaml \
+    --stack-name {{ cookiecutter.project_name }} \
+    --capabilities CAPABILITY_IAM
+```
+
+## Use the SAM CLI to build and test locally
+
+Build your application with the `sam build` command.
+
+```bash
+{{ cookiecutter.project_name }}$ sam build
+```
+
+The SAM CLI installs dependencies defined in `hello_world/requirements.txt`, creates a deployment package, and saves it in the `.aws-sam/build` folder.
+
+Test a single function by invoking it directly with a test event. An event is a JSON document that represents the input that the function receives from the event source. Test events are included in the `events` folder in this project.
+
+Run functions locally and invoke them with the `sam local invoke` command.
+
+```bash
+{{ cookiecutter.project_name }}$ sam local invoke HelloWorldFunction --event events/event.json
```

Comment:
Would this work in your example here?

Reply:
Yes, I've tested sam local invoke in all cases, though usually i don't provide the funciton name - just `sam local invoke --event events/event.json`